src/data.table.h

```diff
@@ -33,6 +33,10 @@ typedef R_xlen_t RLEN;
 #define SIZEOF(x) sizes[TYPEOF(x)]
 #define TYPEORDER(x) typeorder[x]
 
+static char internal_error_buff[128];
```

Comment:
Am I right that with this setup, if your R session produces _more than 1_ `INTERNAL_ERROR` call, and the later call uses less characters than the first, that leftover part will be printed the second time around? Does that matter for an internal error like this?